aptos-move/block-executor/src/view.rs

```diff
@@ -741,6 +732,70 @@ impl<'a, T: Transaction, S: TStateView<Key = T::Key>, X: Executable> LatestView<
         Ok((patched_bytes, mapping.into_inner()))
     }
 
+    fn extract_identifiers_from_value(
+        &self,
+        bytes: &Bytes,
+        layout: &MoveTypeLayout,
+    ) -> anyhow::Result<HashSet<T::Identifier>> {
+        let mapping = TemporaryExtractIdentifiersMapping::<T>::new();
+        let _patched_value =
+            deserialize_and_replace_values_with_ids(bytes.as_ref(), layout, &mapping).ok_or_else(
```

Comment:
This would also generate more identifiers, seems like it's sort of abused here 😁

Reply:
Extract all not create more identifiers. 
Seems like functions are poorly named - they receive a trait,  but are named based on the trait we use. 

Probably better to be deserialize_with_mapping_apply or something